languages/dart/info/arrow_function_expression.md

```diff
@@ -0,0 +1,3 @@
+# Arrow Function Expression
+
+An arrow function expression is a syntactically compact alternative to a regular function expression, although without its own bindings to the `this` or `super` keywords.
```

Comment:
I think this should be mentioned in the transition/from_js.md. Not in the general documentation for arrow functions. The `this` and `super` keywords are only available inside the scope of a class method (including getters/setters/operators/constructors). They are not available to top-level functions.